tickets/src/components/content/NewEventContent.tsx

```diff
@@ -27,6 +27,49 @@ enum PaywallStatus {
   Unknown = 'unknown',
 }
 
+interface EventContentBodyProps {
+  config: {
+    unlockAppUrl: string
+  }
+  event: any
+  paywallStatus: PaywallStatus
+}
+
+export const EventContentBody = ({
```

Comment:
I did this so we could have stories that show the various states (it's pulled from the `EventContent` component below)